runtime/vm/jvminit.c

```diff
@@ -1815,6 +1815,18 @@ IDATA VMInitStages(J9JavaVM *vm, IDATA stage, void* reserved) {
 				}
 			}
 
+			/* -XX commandline option for +/- TransparentHugepage */
+			argIndex = FIND_ARG_IN_VMARGS(EXACT_MATCH, VMOPT_XXNOTRANSPARENT_HUGEPAGE, NULL);
+			argIndex2 = FIND_ARG_IN_VMARGS(EXACT_MATCH, VMOPT_XXTRANSPARENT_HUGEPAGE, NULL);
+			{
+				/* last instance of +/- TransparentHugepage found on the command line wins */
+				if (argIndex2 > argIndex) {
+					j9port_control(J9PORT_CTLDATA_VMEM_ADVISE_HUGEPAGE, 1);
+				} else if (argIndex > argIndex2) {
+					j9port_control(J9PORT_CTLDATA_VMEM_ADVISE_HUGEPAGE, 0);
+				}
```

Comment:
This defaults to not madvising transparent huge pages.  @vijaysun-omr Do you have data indicating which way this should default?

Reply:
ping @vijaysun-omr.  Do you have data on indicating which way madavise for transparent largepages should default?  I suspect the default should be "enabled"